src/pages/student/main/PasswordReset/index.js

```diff
@@ -45,96 +41,76 @@ const PasswordReset = () => {
-                <Form.Label>
-                  <h6 style={{ marginBottom: '0px' }}>Email</h6>
-                </Form.Label>
-                <Controller
-                  control={control}
-                  name="email"
-                  defaultValue=""
-                  render={({ field: { onChange, value, ref } }) => (
-                    <Form.Control
-                      onChange={onChange}
-                      value={value}
-                      ref={ref}
-                      className="cntrs"
-                      type="email"
-                      isInvalid={!!errors?.email}
-                      required
-                      maxLength={50}
-                    />
-                  )}
-                />
-                <Form.Control.Feedback type="invalid">
-                  {errors?.email}
-                </Form.Control.Feedback>
-              </Form.Group>
-
-              <center>
-                <Button
-                  type="submit"
-                  onClick={() => {
-                    setShow(false);
-                  }}
-                  id={style.button}
+            {status === false ? (
+              <div className={style.center} align="start">
+                <center>
+                  <Form.Label>
+                    {' '}
+                    <h4>Password Reset</h4>
+                  </Form.Label>
+                </center>
+                <Form.Group
+                  id={style.Containercentermargin}
+                  className="mb-3"
+                  controlId="Email"
                 >
-                  <span className={style.textbutton}>Send Recovery Link</span>
-                </Button>
-              </center>
+                  <Form.Label>
+                    <h6 className="mb-0">Email</h6>
+                  </Form.Label>
+                  <Controller
+                    control={control}
+                    name="email"
+                    defaultValue=""
+                    render={({ field: { onChange, value, ref } }) => (
+                      <Form.Control
+                        onChange={onChange}
+                        value={value}
+                        ref={ref}
+                        className="cntrs"
+                        type="email"
+                        isInvalid={error}
+                        required
+                        maxLength={50}
+                      />
+                    )}
+                  />
+                  <Form.Control.Feedback type="invalid">
+                    {error}
+                  </Form.Control.Feedback>
+                </Form.Group>
 
-              <center>
-                <div>
-                  <a href="login" className={style.cancel}>
-                    Cancel
-                  </a>
+                <center>
+                  <Button
+                    type="submit"
+                    id={style.button}
+                    className={submitStatus === true ? 'disabled' : ''}
```

Comment:
no need for `=== true`
```suggestion
                    className={submitStatus ? 'disabled' : ''}
```

Reply:
Did you purposely remove disabled={submitStatus} prop? i think its also important